containers/record-linkage/app/main.py

```
@@ -1,12 +1,34 @@
 from fastapi import FastAPI
 from pathlib import Path
 from pydantic import BaseModel, Field
+from contextlib import asynccontextmanager
+import psycopg2
+import os
 
 # from app.config import get_settings
 
 # Read settings immediately to fail fast in case there are invalid values.
 # get_settings()
 
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    dbname = os.getenv("DB_NAME", "testdb")
+    user = os.getenv("DB_USER", "postgres")
+    password = os.getenv("DB_PASSWORD", "pw")
+    host = os.getenv("DB_HOST", "db")
```

I'm a little on the fence here about providing default values. We have generally be trying to have things fail fast. If a user doesn't set the env vars I think we want this service to immediately fail. Thoughts?

---

These are really for local dev, moved them to the docker compose